mobile/lib/models/kya.dart

```diff
@@ -15,30 +15,31 @@ class Kya extends Equatable {
     required this.lessons,
     required this.progress,
     required this.completionMessage,
-    required this.secondaryImageUrl,
-    required this.shareLink,
+    this.shareLink,
   });
 
   final String title;
 
-  @JsonKey(defaultValue: 'You just finished your first Know You Air Lesson')
+  @JsonKey(
+      defaultValue: 'You just finished your first Know You Air Lesson',
+      name: 'completion_message')
```

Comment:
I think we should not have default values. The backend should provide all the data.

Reply:
Done